docs/getting-started/documentation/pull-data-transfer.md

```diff
@@ -34,11 +34,15 @@ When initiating the transfer, select "Custom Transfer Process JSON", and provide
 ```json
 {
   "@type": "https://w3id.org/edc/v0.0.1/ns/TransferRequest",
+  "https://w3id.org/edc/v0.0.1/ns/assetId": "{{ASSET_ID}}",
+  "https://w3id.org/edc/v0.0.1/ns/contractId": "{{AGREEMENT_ID}}",
+  "https://w3id.org/edc/v0.0.1/ns/connectorAddress": "{{COUNTER_PARTY_DSP_ENDPOINT}}",
+  "https://w3id.org/edc/v0.0.1/ns/connectorId": "{{COUNTER_PARTY_ID}}",
```

Comment:
The UI API Wrapper is called with a contract ID. I believe it is extracting this metadata on the backend side and adding it to the Eclipse EDC JSON-LD before the service call.

Reply:
Thanks for this information. I was also able to start the http-pull without the `contractId`.